src/SpeedyTransforms/SpeedyTransforms.jl

```diff
@@ -9,11 +9,16 @@ import FFTW
 import GenericFFT
 import LinearAlgebra
 import Primes
+import Adapt: adapt
 
 # SPEEDYWEATHER MODULES
 using ..LowerTriangularMatrices
 using ..RingGrids
 
+# import SpeedyWeatherCUDAExt
+using SpeedyWeather
+Base.get_extension(SpeedyWeather, :SpeedyWeatherCUDAExt)
```

Comment:
Is this really needed? 

As far as I can see it you only extend functions that are already there in the main module, so it doesn't seem necessary to me.

Reply:
Ahh good catch, left that in accidentally